### DISCUSSION TITLE
Allow nested relational data in button-links interface

### DISCUSSION THREAD

**Discussion by Dominic-Marcelino** [directus/directus#14886] [Feature Requests]
### Describe the Bug

Creating a button that should use nested informations from a relation won't work. 

### To Reproduce

Create the following collections

| plattforms |
|---|
|title|
|link|

| content |
|---|
|id|
|m2o relation to plattforms|

Then try add a button to `content` built of the plattforms.link and the content.id. The platform.link is never fetched.

### What version of Directus are you using?

9.11.1

### What version of Node.js are you using?

N/A

### What database are you using?

MySQL

### What browser are you using?

N/A

### How are you deploying Directus?

N/A

**Comment by azrikahar** [comment]
This is the same situation with #8228, where the nested values of relational fields aren't really available in the form by default. Ideally should be addressed at the same time, but that'd depend on the solution used.

**Comment by Dominic-Marcelino** [comment]
Thanks @azrikahar 

> Ideally should be addressed at the same time, but that'd depend on the solution used.

Does that means, that there's some kind of workaround? 🤔 Or what exactly do you mean with the second part?

**Comment by azrikahar** [comment]
> Does that means, that there's some kind of workaround? 🤔 Or what exactly do you mean with the second part?

Unfortunately I don't think there's any immediate workaround that I can think of for this scenario. 

Based on https://github.com/directus/directus/issues/8228#issuecomment-1157227933 and IIRC there may have been discussions around that recently if I'm not mistaken, my comment is more so just as a little note in case #8228 gets worked on 😄

**Comment by rijkvanzanten** [comment]
@Dominic-Marcelino Currently, the top level item only contains the top level fields so relational m2os are primary keys rather than the nested values. Therefore, nested fields don't "exist" in the context that the button links can see 👍🏻 

Azri mentions that there's a similar situation going on with conditional fields, as they too only support checking against the top level values. 

This is technically a new feature, as the buttons links are only made to support the top level items. I'll move this to a feature request discussion 👍🏻

**Comment by Dominic-Marcelino** [comment]
@rijkvanzanten @azrikahar Thanks for the responses and explanation!

In this case I'll probably create a custom solution in the future.

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
